charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: fluentd-elasticsearch
-version: 11.12.0
+version: 11.12.1
```

Comment:
as this is a new feature minor version should be raised.

Reply:
I indeed raised the minor version instead.